cluster/os-3.9.0-alpha.4/provider.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+image="os-3.9@sha256:a3c66710e0f4d55e81d5b2d32e89c074074cc14b216941818bde0d68cf4b0a12"
+
+source cluster/ephemeral-provider-common.sh
+
+function up() {
+    # Add one, 0 here means no node at all, but in the kubevirt repo it means master-only
+    local num_nodes=${VAGRANT_NUM_NODES-0}
+    num_nodes=$((num_nodes + 1))
+    ${_cli} run --nodes ${num_nodes} --reverse --osp-port 127.0.0.1:8443 --ssh-port 127.0.0.1:2201 --background --registry-port 127.0.0.1:5000 --prefix $PROVIDER --registry-volume kubevirt_registry --base "kubevirtci/${image}"
```

Comment:
We now support multiple nodes. Could you update the line?

Reply:
already did

Reply:
Maybe I missed it.